apps/web/src/services/home-dashboard/modules/all-summary/AllSummary.vue

```diff
@@ -471,7 +492,7 @@ export default {
             state.loading = true;
             await Promise.all([
                 getCount(),
-                getBillingCount(),
+                allReferenceStore.load('costDataSource'),
```

Comment:
question: it doesn't work without this line?

Reply:
Oh it works. I'll delete it.